recipes/capnproto/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir build
+cd build
+cmake -DBUILD_TESTING=OFF -DCMAKE_CXX_FLAGS="-fPIC" -DCMAKE_INSTALL_PREFIX="$PREFIX" ../c++
+
+make
```

Comment:
Yeah, looks like the test suite might be broken based on a [comment]( https://github.com/sandstorm-io/capnproto/issues/227#issuecomment-138741668 ) in the issue you cited. We can probably just leave this alone for now.